### PR TITLE
Introduction update to add clarity about the Material/Geometry compliance of the standard

### PR DESCRIPTION
**Pull request by AsamDiegoSanchez**
## Describe your changes

Further information added in the introduction about how the standard is structured and how the users can be conform with.

## Issue ticket number and link

https://github.com/asam-ev/OpenMATERIAL/issues/169

## Checklist before requesting a review

- [X] I have performed a self-review of my code/documentation.
- [X] My changes generate no new warnings during the documentation generation.

### CODE REVIEW

**Comment by ClemensLinnhoff** [comment]
To  me it was a bit unstructured, because in the first part the different compliance options were already introduced but then there was a spearate section about it. Also, introducing a sub-headline after a text is not allowed according to the editorial guide.

So I took the liberty of restructuring this a little bit. Would this be okay for you @AsamDiegoSanchez ?

**Review by ClemensLinnhoff — APPROVED** [review]

**Review by LudwigFriedmann — COMMENTED** [review]
Just some minor remarks ...

**Review by LudwigFriedmann — CHANGES_REQUESTED** [review]
Sorry I missed to request changes

File: content/general-docs/introduction.adoc [hunk 1]
```diff
@@ -26,16 +26,27 @@ Standardizing the representation of data and attributes, such as scenarios, road
...
+Each chapter addresses specific aspects of the standard, enabling users to adopt and implement the parts most relevant to their needs.
 
-* *Material*: This part includes definitions and file formats for storing and exchanging material properties. These properties can be physical, such as surface roughness, permittivity, and index of refraction, or measured data, like wavelength and angle-dependent reflectance values.
-* *Geometry*: This part contains node hierarchies for different object classes. These structures use a uniform naming scheme and coordinate definitions to enable the exchange of 3D models between various simulation systems.
+* *Material*: This chapter includes definitions and file formats for storing and exchanging material properties. These properties can be physical, such as surface roughness, permittivity, and index of refraction, or measured data, like wavelength and angle-dependent reflectance values.
+* *Geometry*: This chapter contains node hierarchies for different object classes. These structures use a uniform naming scheme and coordinate definitions to enable the exchange of 3D models between various simulation systems.
```
[the remark this answers is not in the document]
Comment (AsamDiegoSanchez) [reply content/general-docs/introduction.adoc:35]: Updated

File: content/general-docs/introduction.adoc [hunk 2]
```diff
@@ -26,16 +26,27 @@ Standardizing the representation of data and attributes, such as scenarios, road
...
 
-These two parts may be used independently.
-For example, a simulation tool may use the material definitions without supporting the geometry part, or the other way around.
-For a full exchange of simulation-ready 3D assets however, both parts must be supported.
+One of the key features of ASAM OpenMATERIAL 3D is its modular approach to compliance.
+This flexibility ensures the standard meets the specific needs of a wide range of applications.
+Users can achieve compliance with the standard in the following ways:
+
+* *Geometry Chapter Compliance*: Users who primarily deal with geometric data, such as 3D models, shapes, or spatial structures, can claim compliance with the Geometry chapter independently.
```
[the remark this answers is not in the document]
Comment (AsamDiegoSanchez) [reply content/general-docs/introduction.adoc:41]: done

**Review by AsamDiegoSanchez — COMMENTED** [review]
Signed-Off by: Diego Sánchez <diego.sanchez@asam.net>

**Pull request merged**